community/index.md

```diff
@@ -83,6 +83,27 @@ cleartax_in_user:
   alt: "ClearTax.in Logo"
 ---
 
+## Engage the community and get support
+
+The Spinnaker OSS project was started at Netflix, and today thrives on the contributions and broad adoption of the public devops community. Here are some ways to engage the community and find help&mdash;join us!
+
+* [Spinnaker community forum](https://community.spinnaker.io)
+
+    Closely monitored by Spinnaker principle developers, this forum is a good place to initiate discussions that might go on for a while and that won't scroll off the screen before you notice them.
+
+* [GitHub](https://github.com/spinnaker)
+
+    The Spinnaker project, with repos for its many services, including [Spinnaker itself](https://github.com/spinnaker/spinnaker) and the [repo for this documentation site](https://github.com/spinnaker/spinnaker.github.io). Come here to contribute to the Spinnaker services and the documentation.
```

Comment:
That main repo you linked to is really a collection of wrapper scripts, glue & testing. I would point that out instead since it's confused people looking through github before. Maybe link to the issue tracker?